README.md

```diff
@@ -18,7 +54,7 @@ An array of resources with links to or the data itself. Includes a resource `nam
 Possible Mediatypes
 
 - `application/geo+json`: GeoJSON with [Mapbox Simple Styles](https://github.com/mapbox/simplestyle-spec)
-- `application/vnd.simplestyle-extended`: GeoJSON with [Extended Simple Styles](https://github.com/cividitech/simplestyle-extended-spec)
+- `application/vnd.simplestyle-extended`: GeoJSON with [Extended Simple Styles](https://github.com/cividi/simplestyle-extended-spec)
 - `application/vnd.mapbox-vector-tile`: Mapbox URI for styled vector tiles
 
```

Comment:
@n0rdlicht could we get at least one mediatype with pure OpenStreetMap?

Reply:
Sure. Other mediatypes are currently not supported by the platform as we have a custom functions per mediatype listed here and would need to add support in the viewer before this becomes useful.